pkg/cmd/pulumi/new.go

```diff
@@ -813,6 +848,102 @@ func saveConfig(stack backend.Stack, c config.Map) error {
 	return saveProjectStack(stack, ps)
 }
 
+func runtimeOptions(ctx *plugin.Context, info *workspace.ProjectRuntimeInfo,
+	main string, opts display.Options, yes, interactive bool, prompt promptForValueFunc,
+) (map[string]interface{}, error) {
+	programInfo := plugin.NewProgramInfo(ctx.Root, ctx.Pwd, main, info.Options())
+	lang, err := ctx.Host.LanguageRuntime(info.Name(), programInfo)
```

Comment:
need to close off this lang plugin before returning from the method, process leak